partiql-spi/src/main/kotlin/org/partiql/spi/Plugin.kt

```diff
@@ -21,12 +21,16 @@ import org.partiql.spi.function.PartiQLFunctionExperimental
 /**
  * A singular unit of external logic.
  */
+@OptIn(PartiQLFunctionExperimental::class)
```

Comment:
Could this remain to be a function level annotation?

Reply:
Also, should we use `@PartiQLFunctionExperimental` instead to propagate the optin requirement?

Reply:
I agree it should be function level. Will update. Thank you